invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsSearch/components/DocumentListEntry/DocumentListEntry.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import { Grid, Item, List } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { DocumentAuthors } from '../../../DocumentsDetails/components/DocumentMetadata/components';
+import { Link } from 'react-router-dom';
+import { FrontSiteRoutes } from '../../../../../../routes/urls';
+import { getCover } from '../../../../config';
+import Truncate from 'react-truncate';
+import { DocumentTags } from '../../../DocumentsDetails/components/DocumentMetadata/components/DocumentTags';
+import isEmpty from 'lodash/isEmpty';
+import { DocumentLanguages } from '../../../DocumentsDetails/components/DocumentMetadata/components/DocumentLanguages';
+
+export default class DocumentListEntry extends Component {
+  constructor(props) {
+    super(props);
+    this.metadata = props.metadata;
+  }
+
+  renderCirculationInfo = meta => {
+    if (meta.circulation.has_items_for_loan > 0 || meta.eitems.total > 0) {
+      return (
+        <List className={'document-availability-wrapper'}>
+          {meta.circulation.has_items_for_loan > 0 ? (
+            <List.Item>
+              <List.Icon color={'green'} name={'check'} />
+              <List.Content>Available for loan</List.Content>
+            </List.Item>
+          ) : null}
+          {meta.eitems.total > 0 ? (
+            <List.Item>
+              <List.Icon name="desktop" />
+              <List.Content>Electronic version</List.Content>
+            </List.Item>
+          ) : null}
+        </List>
+      );
+    }
+    return null;
+  };
+
+  renderImprintInfo = () => {
+    if (!isEmpty(this.metadata.imprints)) {
+      return (
+        <Grid.Column width={4}>
+          <List>
+            <List.Item>
+              <List.Content>
+                <span>Published by: </span>
+                {this.metadata.imprints[0].publisher}
+              </List.Content>
+            </List.Item>
+            <List.Item>
+              {this.metadata.imprints[0].place},{' '}
+              {this.metadata.imprints[0].date}
+            </List.Item>
+          </List>
+        </Grid.Column>
+      );
+    }
+    return null;
+  };
+
+  render() {
+    return (
+      <Item>
+        <Item.Image
+          src={getCover(this.metadata.pid)}
+          size="small"
+          floated="left"
+        />
+        <Item.Content>
+          <Item.Meta>{this.metadata.document_type}</Item.Meta>
+          <Item.Header>
+            <Link to={FrontSiteRoutes.documentDetailsFor(this.metadata.pid)}>
+              {this.metadata.title}
+            </Link>
+          </Item.Header>
+          <Item.Meta>
+            <DocumentAuthors metadata={this.metadata} prefix={'by: '} />
+          </Item.Meta>
+          <Item.Description>
+            <Truncate lines={3}>{this.metadata.abstract}</Truncate>
```

Comment:
We could use `show more` instead of truncate

Reply:
I didn't want to use the `Show more` on the list of the books to make it simple and with not too much controls. And it is a clickbait. Wanna know more? go to the details...